subxt/src/storage/storage_key.rs

```diff
@@ -125,53 +136,28 @@ impl<K: ?Sized> StorageKey for StaticStorageKey<K> {
             }
             .into());
         };
-        *hashers_and_ty_ids = &hashers_and_ty_ids[1..]; // Advance cursor by 1
-        decode_storage_key_from_hash(cursor, hasher, *ty_id, metadata)
-    }
-}
 
-pub fn decode_storage_key_from_hash<K: ?Sized>(
```

Comment:
I removed this specific-to-StaticStorageKey function and made a slightly more general `consume_hash_returning_bytes` one that could then be used in a couple of other places too, to cut down the number of util functions about